### ISSUE TITLE
[Question] boot2docker-experimental.iso creation? (v1.12.x)

### ISSUE THREAD

**Issue by AkihiroSuda** [boot2docker/boot2docker#1192]
https://github.com/boot2docker/boot2docker/releases/download/v1.12.1-rc1/boot2docker-experimental.iso seems actually containing the latest `dockerd`

However, `Dockerfile.experimental` does not seem to update `dockerd`:
https://github.com/boot2docker/boot2docker/blob/v1.12.1-rc1/Dockerfile.experimental#L6-L8

How do you build boot2docker-experimental.iso?

**Comment by yosifkit** [comment]
I checked the build server and it was using a custom Dockerfile similar to my new PR #1193.

**Comment by tianon** [comment]
Even more specifically, it's building via https://github.com/tianon/jenkins-groovy/blob/8f9a123a20e19367a1dcff9d9d7b491548b47454/dsl/tianon/boot2docker.groovy#L53-L79 currently (which after #1193 will be updated to use `Dockerfile.experimental` as-is instead of building it custom at build time).

**Comment by AkihiroSuda** [comment]
ok, thank you
